src/Utility/Xml.php

```diff
@@ -150,6 +150,8 @@ protected static function _loadXml($input, $options)
                 $xml = new DOMDocument();
                 $xml->loadXML($input);
             }
+        } catch (\Throwable $e) {
+            $xml = null;
         } catch (Exception $e) {
             $xml = null;
```

Comment:
```
    /**
     * Parse the input data and create either a SimpleXmlElement object or a DOMDocument.
     *
     * @param string $input The input to load.
     * @param array $options The options to use. See Xml::build()
     * @return \SimpleXmlElement|\DOMDocument
     * @throws \Cake\Utility\Exception\XmlException
     */
```

There is no mention that this will return `null` on any failure, and there is no comment in the code to explain why. Shouldn't this throw the exception? Right now, it silently returns null but no reason given.